src/application.coffee

```diff
@@ -463,8 +467,10 @@ getRemoteState = (uuid, apiKey) ->
 		console.error("Failed to get state for device #{uuid}. #{err}")
 		throw err
 
+# TODO: Actually store and use app.environment and app.config separately
 parseEnvAndFormatRemoteApps = (remoteApps, uuid, apiKey) ->
 	appsWithEnv = _.mapValues remoteApps, (app, appId) ->
+		_.merge(app.environment, app.config)
```

Comment:
I think this can be dropped, along with the:
```coffee
env = _.omit(fullEnv, _.keys(specialActionEnvVars))
env = _.omit env, (v, k) ->
	_.startsWith(k, device.hostConfigEnvVarPrefix)
```
below